src/screens/Login.tsx

```diff
@@ -23,9 +36,9 @@ class Login extends React.Component<any, any> {
                         <p className="message">Already registered? <a href="#">Sign In</a></p>
                     </div>
                     <div className="login-form">
-                        <input type="text" placeholder="username" />
-                        <input type="password" placeholder="password" />
-                        <button onClick={() => this.login()}>login</button>
+                        <input type="text" name="username" placeholder="username" onChange={this.handleChange} />
+                        <input type="password" name="password" placeholder="password" onChange={this.handleChange} />
+                        <button onClick={() => Store.loginUser(this.state.username, this.state.password)}>login</button>
```

Comment:
Needs to grab the user info through MobX

Reply:
Edit: Not mobX, but needs to handle state by using typescript state, and not the normal JS/React State.